kotlin/internal/opts.bzl

```diff
@@ -49,6 +49,13 @@ _KOPTS = {
         ),
         type = attr.bool,
     ),
+    "x_no_optimized_callable_references": struct(
+        args = dict(
+            default = False,
+            doc = "Do not use optimized callable reference superclasses available from 1.4.",
```

Comment:
nit: "Do not use optimized callable reference superclasses. Available from 1.4."